HyperPlay-Gaming/hyperplay-desktop-client
# [UI] Add App Version to Settings

add appversion to settings. remove unused heroic version element


### Testing
1. Navigate to General Settings
2. See the app version at the top of the tab panel

---

Use the following Checklist if you have changed something on the Backend or Frontend:
